### ISSUE TITLE
Missing {pres} message on p2p topic creation.

### ISSUE THREAD

**Issue by freele** [tinode/chat#575]
### Subject of the issue
Describe your issue here.

#### Server-side
- [x] Our own setup:
  * Test setup - Linux inside docker
  * version of Tinode server `0.16.8`, forked and slightly modified - bug fixes and some specific use cases handling. No core logic updates.
  * RethinkDB
  * standalone

#### Client-side
It is reproducible on all the clients we tested
- [x] Tindroid: Android app
- [x] Tinodios: iOS app
  
For simplicity lets go with the testsuite.
- [x] Your own client. Describe it:
  * Transport (gRPC)
  * javascript

### Steps to reproduce
Here is an excerpt from the tests suite
Remove pre-created p2p subs from test data (our test suite uses similar test data)
https://github.com/tinode/chat/blob/ff34443beac639ea4d1b91dee319ad60b88e6549/tinode-db/data.json#L162

Reproduce following logic
```ts
        const [eve, frank] = p2pUsers
        const { stream: streamEve, client: clientEve, user: userEve } = eve
        const { storage: storageFrank } = frank

        const [ctrlEve] = await sendAndWait<pbx.IClientSub>(streamEve, 'sub', { topic: frank.id, getQuery: { what: 'sub' } }, ['ctrl']) as [pbx.IServerCtrl]
        assert.equal(ctrlEve.code, 200)
        await wait(1500)

        console.log(storageFrank.pres)
```
### Expected behaviour
Not empty storage of presence messages.

### Actual behaviour
No presence was received.

If there were pre-created p2p subscription between users, then user2 receives {pres}, but the idea is to invite new user to p2p topic, who does not know about you beforehand.

In other words
`User1` wants to invite `user2` to p2p topic. `User2` does not know about `user1` initially. `User1` creates p2p topic, receives 200 status, but `user2` does not receive anything (exptected - pres message about new p2p topic with `user1`).

If you will not be able reproduce it, please let me know, I'll cleanup and share my testsuite with fully described usecase.

**Comment by or-else** [comment]
Yes, it's a known bug. It affects group topics too.

**Comment by freele** [comment]
@or-else 
Got it, thanks. I did not find it in issues though.

Is there any estimations when it can be fixed? I mean, maybe you guys already know what's the problem. Or is it a complete black box for now?

I found a workaround - for instance, public info updates still arrives, but the logic becomes a bit dirty with that.

**Comment by sanmiguel2019** [comment]
freele, I guess you are are describing the same issue as [here](https://github.com/tinode/chat/issues/574).

**Comment by freele** [comment]
> freele, I guess you are are describing the same issue as [here](https://github.com/tinode/chat/issues/574).
Yes, thanks, I see that it is listed in the problems you described (in details - thanks).
Anyhow, any details or eta's of potential fixes would be appreciated..

**Comment by or-else** [comment]
Duplicate of https://github.com/tinode/chat/issues/574
